app/models/news_article.rb

```diff
@@ -1,6 +1,15 @@
 class NewsArticle < Node
+  has_many :news_distributions
+  has_many :sections, through: :news_distributions, source: :distribution,
+           source_type: 'Section'
+
+  alias_method :publisher, :section
+
   store_attribute :data, :release_date, Date
 
+  content_attribute :short_summary
+  content_attribute :summary
```

Comment:
Are you thinking of shifting this to Node? Or maybe better to mirror the same attribute on GeneralContent? Or - leave it till a later PR...?